vars/build.groovy

```diff
@@ -49,21 +52,27 @@ def call(Map args) {
   }
 }
 
-def createImageStream(imageStreams, namespace) {
-    imageStreams.each { imageStream ->
-        def isName = imageStream.metadata.name
-        def isFound = Utils.shWithOutput(this, "oc get is/$isName -n $namespace --ignore-not-found")
-        if (!isFound) {
-          Utils.ocApply(this, imageStream, namespace)
-        } else {
-          echo "image stream exist ${isName}"
-        }
+def createImageStream(imageStreams, ns) {
+    imageStreams.each { is ->
+      ocApply(this, is, ns)
```

Comment:
thats going to apply the is every time isnit it ?

Reply:
yes in build namespaces

Reply:
Previously pipline were checking the image stream to avoid trigger changes. Now we have fixed those trigger change side effect.